src/main/java/org/squiddev/plethora/integration/astralsorcery/MetaBlocks.java

```diff
@@ -0,0 +1,365 @@
+		public TileIlluminator getExample() {
+			return null;
+			/*TileIlluminator illuminator = new TileIlluminator();
+			setExampleBlockStateWithTile(BlocksAS.blockIlluminator.getDefaultState(), illuminator);
+
+			illuminator.onWandUsed(EnumDyeColor.YELLOW);
+			return illuminator;*/
+		}
+	};
+
+	public static final IMetaProvider<TileAttunementRelay> META_SPECTRAL_RELAY = new BaseMetaProvider<TileAttunementRelay>(
+		"Provides the item in a Spectral Relay"
+	) {
+
+		@Nonnull
+		@Override
+		public Map<String, ?> getMeta(@Nonnull IPartialContext<TileAttunementRelay> context) {
+			// While it also functions as a starlight collector, and is part of attunement and top-tier altar crafting,
+			// the stored item is the only thing that we can easily access.
+			// We _could_ extract the linked BlockPos and check the type of TileEntity there, but... meh.
+			return Collections.singletonMap("item",
+				context.makePartialChild(context.getTarget().getInventoryHandler().getStackInSlot(0)).getMeta());
+		}
+
+		@Nullable
+		@Override
+		public TileAttunementRelay getExample() {
+			return null;
+			/*TileAttunementRelay relay = new TileAttunementRelay();
+			setExampleBlockStateWithTile(BlocksAS.attunementRelay.getDefaultState(), relay);
+
+			relay.getInventoryHandler().insertItem(0, ItemCraftingComponent.MetaType.GLASS_LENS.asStack(), false);
+			return relay;*/
+		}
+	};
+
+
+	public static final IMetaProvider<TileMapDrawingTable> META_STELLAR_REFRACTION_TABLE = new BaseMetaProvider<TileMapDrawingTable>(
+		"Provides the items in a Stellar Refraction Table"
+	) {
+
+		@Nonnull
+		@Override
+		public Map<String, ?> getMeta(@Nonnull IPartialContext<TileMapDrawingTable> context) {
+			Map<String, Object> out = new HashMap<>(2);
+			TileMapDrawingTable target = context.getTarget();
+
+			TypedLuaObject<MetaWrapper<ItemStack>> lensStack = ContextHelpers.wrapStack(context, target.getSlotGlassLens());
+			if (lensStack != null) {
+				out.put("infusedGlass", lensStack);
+			}
+
+			TypedLuaObject<MetaWrapper<ItemStack>> inputStack = ContextHelpers.wrapStack(context, target.getSlotIn());
+			if (inputStack != null) {
+				out.put("processingSlot", inputStack);
+			}
+
+			return out;
+		}
+
+		@Nullable
+		@Override
+		public TileMapDrawingTable getExample() {
+			return null;
+			/*TileMapDrawingTable table = new TileMapDrawingTable();
+			setExampleBlockStateWithTile(BlocksAS.drawingTable.getDefaultState(), table);
+
+			table.addParchment(32);
+			table.putGlassLens(new ItemStack(ItemsAS.infusedGlass));
+
+			return table;*/
+		}
+	};
+
+	//TODO Move this to a better location for utilization in other examples
+	// Also, come up with a better name...
+	//Not currently used, as the examples for Astral tiles are disabled, but likely to be important if we
+	// add support for more mods that require the BlockState as well as the TileEntity for examples.
+	public static void setExampleBlockStateWithTile(@Nonnull IBlockState state, @Nonnull TileEntity tile) {
```

Comment:
Also remove this for now. Hopefully we'll have fixed the underlying issue before this is needed again.